File: GO/euler006.go
package main

import (
	"fmt"
	"math"
)

func main() {
	fmt.Println(solve())
}

func solve() int {
	LIMIT := 100
	sqos := 0
	sosq := 0

	for i := 1; i <= LIMIT; i++ {
		sosq += int(math.Pow(float64(i), 2))
		sqos += i
	}
	sqos = int(math.Pow(float64(sqos), 2))
	return int(math.Abs(float64(sqos - sosq)))
}

File: C#/euler006.cs
using System;

namespace ProjectEuler {
    class euler006 {
        static void Main() {
            Console.WriteLine(Solve());
        }

        static int Solve() {
            int LIMIT = 100;
            int sqos = 0;
            int sosq = 0;
            int res = 0;

            for (int i = 1; i <= LIMIT; i++) {
                sosq += i*i;
                sqos += i;
            }
            sqos *= sqos;
            res = sqos-sosq;

            if (res < 0) {
                res *= -1;
            }

            return res;

        }
    }
}